src/connectedk8s/azext_connectedk8s/azext_metadata.json

```
@@ -1,5 +1,5 @@
 {
     "name": "connectedk8s",
-    "version": "1.2.1",
-    "azext.minCliCoreVersion": "2.16.0"
+    "version": "1.3.1",
+    "azext.minCliCoreVersion": "2.30.0"
```

The extension version is set in setup.py, so there is no need to add a `version` here
```suggestion
    "azext.minCliCoreVersion": "2.30.0"
```

---

correct. hence removed